shell/config/router/navigation-guards/load-initial-settings.js

```diff
@@ -1,13 +1,15 @@
 import { fetchInitialSettings } from '@shell/utils/settings';
 
 export function install(router, context) {
-  router.beforeEach((from, to, next) => loadInitialSettings(from, to, next));
+  router.beforeEach((to, from, next) => loadInitialSettings(to, from, next, context));
 }
 
-export async function loadInitialSettings(from, to, next) {
+export async function loadInitialSettings(to, from, next, { store }) {
   try {
-    await fetchInitialSettings();
```

Comment:
This was never actually doing the heavy lifting. All the extra fetchIninitialSettings were actually doing the work. I fixed this and removed the others.

Reply:
In other words, we previously weren't fetching settings without a reference to the store?